### PR TITLE
Add a search box to find songs and songs.json file

### PR DESCRIPTION
**Pull request by sonnynomnom**
search

- Added a `songs.json` file.
- Added a search box to find songs using the JSON file.
- Also edited the keyboard shortcut to change the background from <kbd>space</kbd> to <kbd>shift</kbd>+<kbd>space</kbd> because it was messing up with the search box when I type any query with space in it lol.

### CODE REVIEW

File: public/js/songs.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  // console.log(filteredsongs);
+});
+
+
+const loadsongs = async () => {
+  try {
+    const res = await fetch('./songs.json');
+    songs = await res.json();
+    displaysongs(songs);
+    console.log(songs);
```
Comment (codecaaron) [inline public/js/songs.js:32]: 🔪

File: public/js/songs.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    return (
+      song.title.toLowerCase().includes(searchString) || 
+      song.artist.toLowerCase().includes(searchString) ||
+      song.album.toLowerCase().includes(searchString)
+    );
+  });
+  
+  displaysongs(filteredSongs);
+  // console.log(filteredsongs);
```
Comment (codecaaron) [inline public/js/songs.js:23]: 🔪 
```suggestion
```

File: public/js/songs.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+console.log(searchBar);
+
+// After an user type a character in the searchbox
+searchBar.addEventListener('keyup', (e) => {
+  const searchString = e.target.value.toLowerCase();
+
+  const filteredSongs = songs.filter( song => {
+    console.log(song.title);
+    console.log(song.artist);
```
Comment (codecaaron) [inline public/js/songs.js:13]:
```suggestion
```

File: public/js/songs.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+    return (
+      song.title.toLowerCase().includes(searchString) || 
+      song.artist.toLowerCase().includes(searchString) ||
+      song.album.toLowerCase().includes(searchString)
```
Comment (codecaaron) [inline public/js/songs.js:18]: ⛳ - dynamically check every key for the search string and if you refactor or add new fields it won't need to be updated.
```suggestion
return Object.values(song).some(value => typeof value === 'string' ? value.toLowerCase().includes(searchString) : false)
```

File: public/js/songs.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    songs = await res.json();
+    displaysongs(songs);
+    console.log(songs);
+  } catch (err) {
+    console.error(err);
+  }
+};
+
+const displaysongs = (songs) => {
+  const htmlString = songs
```
Comment (codecaaron) [inline public/js/songs.js:39]: Hot damn :) look at you

**Review by codecaaron — APPROVED** [review]
Very cool

File: public/js/songs.js (hunk 1 above)
Comment (sonnynomnom) [reply public/js/songs.js]: deleted!

File: public/js/songs.js (hunk 5 above)
Comment (sonnynomnom) [reply public/js/songs.js:32]: 😬

**Comment by sonnynomnom** [comment]
Merged! @codecaaron

**Pull request merged**